packages/helper-grammar-regex-collection/index.js

```diff
@@ -131,6 +131,21 @@ export const HTML_IMPORT = regex`
   href=${captureQuotedWord}>
 `;
 
+// handles the case the HTML script tag has a type with it
+const typeRegex = regex`
+  (\stype=['"][^'"\s]+['"])?
+`;
+
+export const HTML_SCRIPT_IMPORT = regex`
+  ^\s*
+  <script
+  ${typeRegex}
+  \s
+  src=${captureQuotedWord}
+  ${typeRegex}
```

Comment:
Do we need `typeRegex` at all? I'm thinking we could simplify the code and support the presence of other attributes by replacing both instances of `${typeRegex}` with `[^>]*` so anything that's not the end of the tag is matched.

Reply:
that's a really good point.